src/NuGet.Core/NuGet.Packaging/Strings.resx

```diff
@@ -165,4 +165,10 @@
   <data name="InvalidNuspecEntry" xml:space="preserve">
     <value>The nuspec contains an invalid entry '{0}' in package '{1}' .</value>
   </data>
+  <data name="FailToWritePackagesConfig" xml:space="preserve">
+    <value>Fail to write packages.config as XML file due to exception: {0}</value>
```

Comment:
Failed to write `packages.config` while accessing `{0}` : {1}

{0} will be the full path of what is being actually written, and {1} the exception message. Make the try/catch cover that